Tests/src/PublicAPITests/MaterialTests.cpp

```diff
@@ -304,7 +319,7 @@ CSP_PUBLIC_TEST(CSPEngine, MaterialTests, GetInvalidMaterialTest)
 
 	// Attempt to get an invalid material
 	GLTFMaterial FoundMaterial;
-	GetMaterial(AssetSystem, "InvalidAssetCollectionId", "InvalidAssetId", FoundMaterial);
+	GetMaterial(AssetSystem, "InvalidAssetCollectionId", "InvalidAssetId", FoundMaterial, csp::systems::EResultCode::Failed);
```

Comment:
[Nit] Maybe `ExpectGetMaterial` or something that conveys that this is also doing asserts?

Reply:
Agreed. This is a pattern we follow with all of our tests. We should change this when we refactor

Reply:
Wish it was easier to make little issues based off stuff like this, I'm the first to admit the hurdle to doing it often leads to little stuff like this being lost forever and never changing.